homeassistant/components/climate/ecobee.py

```diff
@@ -382,13 +383,3 @@ def hold_preference(self):
             return default
         else:
             return 'nextTransition'
-
-    # Sleep mode isn't used in UI yet:
```

Comment:
Not sure what you mean by "toggle switch" for home or away mode, assuming you are referring to hold_mode. Obviously you would have an input selector in your UI as you are dealing with a choice between a number of alternatives, see the screen shot from my UI...
![screenshot from 2017-01-29 08-11-46](https://cloud.githubusercontent.com/assets/24535446/22404641/b5eed7ba-e5fa-11e6-8c4c-ac2e91fe886a.png)